### ISSUE TITLE
"This is taking longer than usual..." message does not get read out from a screen reader (iOS only)

### ISSUE THREAD

**Issue by knguyenBC** [bcgov/bc-wallet-mobile#565]
When verifying a credential, the “This is taking longer than usual…” message does not get read out on iOS, whereas on Android it does.

**Comment by jleach** [comment]
Tested with the Faber demo so I could naturally trigger the delay. On iOS the screen reader read the "This is taking longer..." message when touched or swiped to it. The black box in the screen shot below shows it identifying the text it was reading at the time:

![Screenshot 2023-05-31 at 3.49.41 PM.png](https://images.zenhubusercontent.com/5c2fa1b7ce4f66381cd8d02d/60b55968-f6f6-416c-8c5f-a1826e63c3e2)

**Comment by nodlesh** [comment]
@jleach @knguyenBC Correct me if I'm wrong Kim, but I believe when text is changed on a screen like this, you should not have to tap it to have it read, I think it is suppose to be read when something new appears. Maybe it gets autofocus when/if it appears?

**Comment by knguyenBC** [comment]
That is correct. I'm not sure what role/label gets added to it but this message should be read out when it appears.

**Comment by nodlesh** [comment]
Moving this back to Estimated Ready.

**Comment by knguyenBC** [comment]
Is there a way to force that message to appear? Loading is too fast now.

**Comment by knguyenBC** [comment]
behaves as expected. tested build 1075 on iOS and Android.
